packages/vue/src/components/organisms/SfHero/SfHero.js

```diff
@@ -0,0 +1,98 @@
+import Vue from "vue";
+
+import SfHeroItem from "./_internal/SfHeroItem.vue";
+import SfArrow from "../../atoms/SfArrow/SfArrow.vue";
+import SfBullets from "../../atoms/SfBullets/SfBullets.vue";
+
+import Glide from "@glidejs/glide";
+
+const BULLETS_POSITION = ["top", "bottom"];
+
+Vue.component("SfHeroItem", SfHeroItem);
+
+export default {
+  name: "SfHero",
+  data() {
+    return {
+      glide: null,
+      defaultOptions: {
+        type: "slider",
+        rewind: true,
+        autoplay: 5000,
+        perView: 1
+      }
+    };
+  },
+  props: {
+    /** Carousel options like glide.js (https://glidejs.com/docs/) */
+    options: {
+      type: Object,
+      default: () => ({})
+    },
+    /** Items to display in SfHero component */
+    heroItems: {
+      type: Array,
+      default: () => []
+    },
+    /** Position of the bullets, either 'top' or 'bottom' */
+    bulletsPosition: {
+      type: String,
+      default: ""
+    }
```

Comment:
what about `sliderOptions`, `current` and `items` (not sure about `items`) as prop names?